contrib/repairStatus.py

```diff
@@ -0,0 +1,199 @@
+from __future__ import print_function
```

Comment:
As per my comment about scope creep below, adding `unicode_literals` is another good candidate for later improvement, probably not something you should try to do before landing this.